src/components/navigation/Navigation.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+const Navigation = () => {
+  return (
+    <nav className="nav">
+      <Link to="/">Home</Link>
+      <Link to="/articles">Articles</Link>
```

Comment:
Great use of link here 👍 
It would be nice to see some more on your nav bar when you get chance